src/main/java/com/flightstats/hub/config/binding/ClusterHubBindings.java

```diff
@@ -63,7 +63,7 @@ protected void configure() {
         bind(AppUrlCheck.class).asEagerSingleton();
 
         bind(SpokeDecommissionManager.class).asEagerSingleton();
-        bind(SpokeTtlEnforcer.class).asEagerSingleton();
+        bind(SpokeTtlEnforcer.class);
```

Comment:
I think we can use implicit/JIT binding for this class unless we start using one for read and one for write (in which case I think we could bind them as eager singletons).